controller/frontend/src/Controller/Frontend/Customer/Standard.php

```diff
@@ -92,8 +96,16 @@ public function add( array $values ) : Iface
 			$this->item->setCode( $code );
 		}
 
+		if ( $oldPassword = $values['customer.oldpassword'] ?? null) {
+		    $confirmed = $values['customer.newpassword'] === $values['customer.confirmnewpassword'];
+		    $isNew = $values['customer.newpassword'] !== $values['customer.oldpassword'];
+		    if ($this->item->verifyPassword($oldPassword) && $confirmed && $isNew) {
```

Comment:
This method isn't available in the customer item yet

Reply:
https://github.com/aimeos/aimeos-core/pull/250

i forgot the aimeos-core pr